File: data/code_pairs/fortran/1035.f90
module m1
   implicit none
contains
   subroutine bar
      print *, "bar executed"
   end subroutine bar

   subroutine bar_none(i)
      integer :: i
      print *, "bar_none executed with i =", i
   end subroutine bar_none

   subroutine none_bar(x)
      real :: x
      print *, "none_bar executed with x =", x
   end subroutine none_bar
end module m1

File: data/code_pairs/cpp/1035.cpp
#include <iostream>
using namespace std;

class m1 {
public:
    void bar() {
        cout << "bar called" << endl;
    }

    void bar_none(int i) {
        cout << "bar_none called with i = " << i << endl;
    }

    void none_bar(float x) {
        cout << "none_bar called with x = " << x << endl;
    }
};

int main() {
    m1 instance;
    instance.bar();
    instance.bar_none(10);
    instance.none_bar(5.5f);

    cout << "All C++ tests passed." << endl;

    return 0;
}